components/hierarchical-view/hierarchical-view-mixin.js

```diff
@@ -80,13 +80,6 @@ export const HierarchicalViewMixin = superclass => class extends superclass {
 
```

Comment:
Changed this back to how it was before https://github.com/BrightspaceUI/core/pull/287/commits/ea0a8c511e8495cc517b3f8b8cac6c9c6e87a0b6 (it says firefox but it applied to edge). This code was causing a test failure in legacy edge and the problem that it was added for seems to be fixed.